Rakefile

```diff
@@ -136,6 +137,35 @@ nugets_pack :create_nuget_quick => [:versioning, 'build/pkg'] do |p|
   end
 end
 
+task :paket => [:versioning, 'build/pkg'] do
+   puts "starting"
+   files = FileList['src/**/*.fsproj'].exclude(/Tests/)
+   files.each do |f|
+    p = Albacore::Project.new f
+    m = "type file
```

Comment:
If you want to use quotes in the string, you can also start the string with `%{` and end with `}`, FYI